TUTORIAL.rst

```diff
@@ -0,0 +1,59 @@
+PDFmorph Tutorial
+=================
+
+Welcome! This will be a quick tutorial to accquaint users with PDFmorph
+and some of what it can do. 
+
+As we described in the README and installation instructions, please make
+sure that you are familiar with working with your command line terminal
+before using this application.
+
+After all of the necessary software is installed, activate your PDFmorph-
+equipped conda environment and change into a directory containing at
+least two PDF files. If you need some test files to help you get started,
+check `here<https://github.com/diffpy/diffpy.pdfmorph/tree/master/tests/testdata>`_.
+
+Once you have two PDF files, run the command ::
+
+	pdfmorph <testfile 1> <testfile 2>
+
+and observe. PDFmorph should pop up a graph of your PDF data "morphed"
+together, with a difference curve underneath.
+
+To see the difference curve working, run ::
+
+	pdfmorph <testfile 1> <testfile 1>
+
+and see the difference curve completely flattened. 
+
+
+Additional PDFmorph Features
+----------------------------
+
+Finally, see that by running ::
+
+	pdfmorph --help
+
+a list of additional modifications that you can run in your PDFmorph
+program is generated. 
+
+You can add any of these "options" by placing them before your two 
+filenames. ::
+
+	pdfmorph [options] <testfile 1> <testfile 2>
```

Comment:
honestly, I think I prefer introducing these things in a step-by-step way.  You can use rhetorical tricks like

type ``pdfmorph --scale=0.5 ni.gr``

It is also a good idea to have them do some things wrong, so if tehy try and morph a higher temperature dataset onto a lower one, what happens?

Reply:
Ok, I'll try to make it more step-by-step next time around (currently trying to figure out the style sheet). So far, I've been using PDF data provided in the repo (tests/testdata) so I don't really know which are higher and lower temps at this point. Could you send me some data that's more clearly marked for temp so I can see for myself too?